pkg/announcements/types.go

```diff
@@ -10,6 +10,9 @@ func (at Kind) String() string {
 }
 
 const (
+	// ProxyUpdate is the event kind used to trigger an update to subscribed proxies
+	ProxyUpdate Kind = "proxy-update"
```

Comment:
Is this functionally different from `ScheduleProxyBroadcast`?

Reply:
ScheduleProxyBroadcast will be deprecated.